source/d3d10/d3d10_device.cpp

```diff
@@ -48,11 +48,13 @@ bool D3D10Device::save_depth_texture(ID3D10DepthStencilView *pDepthStencilView,
 	texture->GetDesc(&desc);
 
 	// Check if aspect ratio is similar to the back buffer one
-	const float screen_aspect_ratio = float(runtime->frame_width()) / float(runtime->frame_height());
+	const float width_factor = desc.Width != runtime->frame_width() ? float(runtime->frame_width()) / desc.Width : 1.0f;
```

Comment:
Why this? If `desc.Width == frame_width`, then result is `1.0f`, so the branch makes no sense?

Reply:
Yes, it's right. Fixed